### PR TITLE
Widget exists

### PR DESCRIPTION
**Pull request by iamdeaneyelid**

### CODE REVIEW

**Comment by iamdeaneyelid** [comment]
@jonsherrard @KimberleyCook @ecoad

File: custom-fields/social-overrides.php [hunk 1]
```diff
@@ -117,7 +117,14 @@
       array (
         'param' => 'post_type',
         'operator' => '==',
-        'value' => 'page',
```
Comment (jonsherrard) [inline custom-fields/social-overrides.php]: Do we want to lose social-overrides from the page post type? Could be useful...

Comment (iamdeaneyelid) [inline custom-fields/social-overrides.php]: Let's put back in

File: custom-fields/WidgetLoader.php [hunk 2]
```diff
@@ -93,9 +92,9 @@ protected static function traversePlugins() {
     if ( ! function_exists( 'get_plugins' ) ) {
       require_once ABSPATH . 'wp-admin/includes/plugin.php';
     }
-    $all_plugins = get_option('active_plugins');
+    $all_plugins = get_plugins();
```
Comment (jonsherrard) [inline custom-fields/WidgetLoader.php]: What's the difference between these to APIs?

Comment (iamdeaneyelid) [inline custom-fields/WidgetLoader.php]: I have no idea. The change happened when I merged...

File: custom-fields/WidgetLoader.php [hunk 3]
```diff
@@ -93,9 +92,9 @@ protected static function traversePlugins() {
...
     // The keys in $all_plugins is the file path for plugin.
-    foreach($all_plugins as $file){
+    foreach($all_plugins as $file => $obj){
```
Comment (jonsherrard) [inline custom-fields/WidgetLoader.php]: Did we need to make `$obj` available? It's not used anywhere else in the file.

File: custom-fields/article-images.php [hunk 4]
```diff
@@ -0,0 +1,119 @@
+<?php
```
Comment (jonsherrard) [inline custom-fields/article-images.php]: Do we need article-images? I think we deleted it from the develop repo?

**Comment by iamdeaneyelid** [comment]
Closed

**Pull request closed without merging**